app/src/main/java/org/sil/bloom/reader/DownloadsView.java

```diff
@@ -446,6 +449,12 @@ private void handleDownloadComplete(String downloadDestPath, long downloadId) {
         String fileName = DownloadsView.getFileNameFromUri(downloadDestPath);
         File dest = new File(BookCollection.getLocalBooksDirectory(), fileName + ".bloompub");
         IOUtilities.copyFile(source.getPath(), dest.getPath());
+        Properties props = new Properties();
+        BloomFileReader reader = new BloomFileReader(mContext, dest.getPath());
+        props.putValue("bookInstanceId", reader.getStringMetaProperty("bookInstanceId", ""));
+        props.putValue("title", reader.getStringMetaProperty("title", ""));
+        props.putValue("originalTitle", reader.getStringMetaProperty("originalTitle", ""));
```

Comment:
Do we have easy access to the parse ID? I think we will want that to easily tie to the notifications later.
Plus, while there is a one-to-one correspondence with bookInstanceId in the ideal world, there isn't necessarily in the real world.

I also wondered if we will want anything about the language. But maybe that's too complicated...
In theory, knowing the book, we know the languages in the book, but that doesn't tell us which language the user was searching within.
I guess if we have easy access to the language they are searching in, we should add it, but my guess is we don't have that.